internal/rbd/nodeserver.go

```diff
@@ -188,6 +201,11 @@ func populateRbdVol(
 	// get rbd image name from the volume journal
 	// for static volumes, the image name is actually the volume ID itself
 	if isStaticVol {
+		if req.GetVolumeContext()["migration"] == "true" {
+			// if migration static volume, use imageName as volID
+			rv.RbdImageName = req.GetVolumeContext()["imageName"]
+			volID = rv.RbdImageName
```

Comment:
why not send volID as the rbdImage name itself from the kubelet instead of doing a hack here? are you planning to add support for image deletion also?

Reply:
Yep, exactly , thats the plan to add deletion but only for migration static volume and not for other static volume (atleast in first version) :+1: